frontend/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "sudan-bot-frontend",
+    "version": "0.0.1",
+    "description": "Dashboard for TheSuperGamer20578/Sudan-bot",
+    "repository": "https://github.com/TheSuperGamer20578/Sudan-bot/tree/master/frontend",
+    "author": "oworangix",
```

Comment:
Can you be slightly more serious?

Reply:
my npm username is literally oworangix

Reply:
can you send link to npm?

Reply:
https://www.npmjs.com/~oworangix i have...nothing :(